pkg/cmd.go

```diff
@@ -21,10 +21,14 @@ func ParseFlags(progname string, args []string) (*Config, string, error) {
 	flags.IntVar(&conf.Draft, "draft", 2020, "Draft version (4, 6, 7, 2019, or 2020)")
 	flags.IntVar(&conf.Indent, "indent", 4, "Indentation spaces (even number)")
 	flags.Var(&conf.NoAdditionalProperties, "noAdditionalProperties", "Default additionalProperties to false for all objects in the schema")
+
 	flags.Var(&conf.Bundle, "bundle", "Bundle referenced ($ref) subschemas into a single file inside $defs")
 	flags.Var(&conf.BundleWithoutID, "bundleWithoutID", "Bundle without using $id to reference bundled schemas, which improves compatibility with e.g the VS Code JSON extension")
 	flags.StringVar(&conf.BundleRoot, "bundleRoot", "", "Root directory to allow local referenced files to be loaded from (default current working directory)")
 
+	flags.StringVar(&conf.K8sSchemaURL, "k8sSchemaURL", "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/refs/heads/master/{{ .K8sSchemaVersion }}/", "URL template used in $ref: $k8s/... alias")
```

Comment:
new flags need to be documented in `README.md` unser [Usage](https://github.com/losisin/helm-values-schema-json?tab=readme-ov-file#usage) and add example in `.schema.yaml`

Reply:
Changed in 5b63026